docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -341,122 +325,260 @@ GET /sample-01*/_search
+Roll over the TSDS using the <<indices-rollover-index,rollover API>>:
 
 [source,console]
 ----
-PUT /sample-01/_block/write
+POST /my-data-stream/_rollover/
 ----
 // TEST[continued]
 
-And now, you can use the <<indices-downsample-data-stream,downsample API>> to
-downsample the index, setting the time series interval to one hour:
+Copy the name of the `old_index` from the response. In the following steps,
+replace the index name with that of your `old_index`.
+
+The old index needs to be set to read-only mode. Run the following request:
 
 [source,console]
 ----
-POST /sample-01/_downsample/sample-01-downsample
+PUT /.ds-my-data-stream-2023.07.26-000001/_block/write
+----
+// TEST[skip:We don't know the index name at test time]
+
+Next, use the <<indices-downsample-data-stream,downsample API>> to downsample
+the index, setting the time series interval to one hour:
+
+[source,console]
+----
+POST /.ds-my-data-stream-2023.07.26-000001/_downsample/.ds-my-data-stream-2023.07.26-000001-downsample
 {
   "fixed_interval": "1h"
 }
 ----
-// TEST[continued]
+// TEST[skip:We don't know the index name at test time]
+
+Now you can <<modify-data-streams-api,modify the data stream>>, and replace the
+original index with the downsampled one:
 
-Finally, delete the original index:
+[source,console]
+----
+POST _data_stream/_modify
+{
+  "actions": [
+    {
+      "remove_backing_index": {
+        "data_stream": "my-data-stream",
+        "index": ".ds-my-data-stream-2023.07.26-000001"
+      }
+    },
+    {
+      "add_backing_index": {
+        "data_stream": "my-data-stream",
+        "index": ".ds-my-data-stream-2023.07.26-000001-downsample"
+      }
+    }
+  ]
+}
+----
+// TEST[skip:We don't know the index name at test time]
+
+You can now delete the old backing index:
 
 [source,console]
 ----
-DELETE /sample-01
+DELETE .ds-my-data-stream-2023.07.26-000001
 ----
-// TEST[continued]
+// TEST[skip:We don't know the index name at test time]
 
 [discrete]
 [[downsampling-manual-view-results]]
 ==== View the results
 
-
-Re-run your search query (note that when querying downsampled indices there are <<querying-downsampled-indices-notes,a few nuances to be aware of>>):
+Re-run the earlier search query (note that when querying downsampled indices
+there are <<querying-downsampled-indices-notes,a few nuances to be aware of>>):
 
 [source,console]
 ----
-GET /sample-01*/_search
+GET /my-data-stream/_search
 ----
-// TEST[continued]
+// TEST[skip:Because we've skipped the previous steps]
 
-In the query results, notice that the number of hits has been reduced to only 288
-documents. As well, for each time series metric statistical representations have
-been calculated: `min`, `max`, `sum`, and `value_count`.
+The TSDS with the new downsampled backing index contains just one document that
+includes the `min`, `max`, `sum`, and `value_count` statistics based off of the
+original sampled metrics:
```

Comment:
This is true for gauges...Maybe say that for a counter we would just see its last value. Also maybe we should highlight that the type of the field is an aggregate now `aggregate_metric_double` instead of the original type.

Reply:
Thanks. Addressed in https://github.com/elastic/elasticsearch/pull/97976/commits/159bdc05c8a31b6ebfa927d96deb1c2c7dc10e4a and https://github.com/elastic/elasticsearch/pull/97976/commits/4ce8a82ff464161e9013962a32ae28ac38a17b6e